sycl/include/CL/sycl/detail/pi.h

```diff
@@ -456,7 +456,7 @@ pi_context piContextCreate( // TODO: change interface to return error code inste
     size_t       cb,
     void *       user_data),
   void *            user_data,
-  pi_result *       result);
+  pi_context *       retcontext);
```

Comment:
```suggestion
  pi_context *       ret_context);
```
Just a minor change to fit the naming convention for the rest of the return values. Very pedantic comment so feel free to ignore it!

Reply:
no worries. 
Changed in the new commit.